scholia/app/templates/base.html

```diff
@@ -86,6 +86,18 @@
 	 }
 	 catch(e) { }
 
+	 console.log(item.claims.P6782[0])
```

Comment:
Why are there this console log message. Isn't it only for debugging?

Reply:
Yes, I guess it would. You did not want that, right? I'll remove it.

Reply:
Yes, we should not have console.logs here. Unless there is something wrong.

Reply:
sorry, wanted to remove that one already. Done now.